pkg/blobplugin/main.go

```diff
@@ -26,45 +26,37 @@ import (
 	"strings"
 
 	"sigs.k8s.io/blob-csi-driver/pkg/blob"
+	"sigs.k8s.io/blob-csi-driver/pkg/util"
 
 	"k8s.io/component-base/metrics/legacyregistry"
 	"k8s.io/klog/v2"
 )
 
+var driverOptions blob.DriverOptions
 var (
-	endpoint                               = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
```

Comment:
it's better putting all flags in one place in main.go, that's easier to managed in long term